challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.collect { |item| item.capitalize }
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.select { |item| item == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+    input.reject { |item| item == "dog" }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.count
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.slice(0, 2)
```

Comment:
Agreed! I actually didn't see that method at first (no pun intended). Is that just an alias for take?